### PR TITLE
Add all tba methods to Requester trait

### PR DESCRIPTION
**Pull request by WaffleLapkin**

### CODE REVIEW

File: src/bot/api.rs [hunk 1]
```diff
@@ -1707,19 +1707,921 @@ impl Bot {
...
+
+    type SetWebhook = JsonRequest<payloads::SetWebhook>;
+
+    fn set_webhook<U, A>(&self, url: U, allowed_updates: A) -> Self::SetWebhook
+    where
+        U: Into<String>,
+        A: IntoIterator<Item = crate::types::AllowedUpdate>,
+    {
+        Self::SetWebhook::new(self.clone(), payloads::SetWebhook::new(url, allowed_updates))
+    }
```
Comment (Hirrolot) [inline src/bot/api.rs:1724]: Don't you think gathering associated types together (and the same for functions) would make the code cuter?

File: src/requests/requester.rs [hunk 2]
```diff
@@ -12,18 +16,686 @@ use crate::{
...
+    /// For telegram documentation see [`GetUpdates`]
+    fn get_updates(&self) -> Self::GetUpdates where;
+
+    type SetWebhook: Request<Payload = SetWebhook, Err = Self::Err>;
+
+    /// For telegram documentation see [`SetWebhook`]
+    fn set_webhook<U, A>(&self, url: U, allowed_updates: A) -> Self::SetWebhook
+    where
+        U: Into<String>,
+        A: IntoIterator<Item = AllowedUpdate>;
```
Comment (Hirrolot) [inline src/requests/requester.rs:34]: The same suggestion with gathering types and functions.

**Review by Hirrolot — CHANGES_REQUESTED** [review]

File: src/bot/api.rs (hunk 1 above)
Comment (WaffleLapkin) [reply src/bot/api.rs:1724]: I don't think so b/c the types and the methods are highly related to each other.

File: src/bot/api.rs [hunk 3]
```diff
@@ -1707,19 +1707,921 @@ impl Bot {
...
+
+    fn delete_webhook(&self) -> Self::DeleteWebhook where {
+        Self::DeleteWebhook::new(self.clone(), payloads::DeleteWebhook::new())
+    }
+
+    type GetWebhookInfo = JsonRequest<payloads::GetWebhookInfo>;
+
+    fn get_webhook_info(&self) -> Self::GetWebhookInfo where {
+        Self::GetWebhookInfo::new(self.clone(), payloads::GetWebhookInfo::new())
+    }
```
Comment (Hirrolot) [inline src/bot/api.rs:1736]: Empty `where` blocks.
May be also somewhere else.

**Review by Hirrolot — CHANGES_REQUESTED** [review]

File: src/bot/api.rs (hunk 3 above)
Comment (WaffleLapkin) [reply src/bot/api.rs:1736]: Huh, I thought rustfmt handles this...

**Review by Hirrolot — APPROVED** [review]

**Pull request merged**